### ISSUE TITLE
AWS Athena usage?

### ISSUE THREAD

**Issue by lynnlangit** [open-guides/og-aws#405]
I am doing a bake-off between AWS [Athena](http://docs.aws.amazon.com/athena/latest/ug/what-is.html) and Google BigQuery for a client.  I am wondering about customer usage of Athena.  Is anyone out these using it in production yet?  If so, should I start a small section on Athena for this og-aws?

**Comment by ThanosBaskous** [comment]
@lynnlangit Thanks for noting the absence of Athena. Another member of my team recently used Athena for ad-hoc analysis of ELB logs and the experience seemed like it was good. I'd definitely be supportive of your creating a section in the guide.

**Comment by QuinnyPig** [comment]
Absolutely. Any chance your bake-off produced anything we can include / link against, @lynnlangit?
